website/docs/d/acm_certificate.html.markdown

```diff
@@ -38,7 +38,7 @@ data "aws_acm_certificate" "rsa_4096" {
 ## Argument Reference
 
 * `domain` - (Required) The domain of the certificate to look up. If no certificate is found with this name, an error will be returned.
-* `key_types` - (Optional) A list of key algorithms to filter certificates. By default, ACM does not return all certificate types when searching. Valid values are `RSA_1024`, `RSA_2048`, `RSA_4096`, `EC_prime256v1`, `EC_secp384r1`, and `EC_secp521r1`.
+* `key_types` - (Optional) A list of key algorithms to filter certificates. By default, ACM does not return all certificate types when searching. Valid values are `RSA_1024`, `RSA_2048`, `RSA_3072`, `RSA_4096`, `EC_prime256v1`, `EC_secp384r1`, and `EC_secp521r1`.
```

Comment:
As values would get automatically updated as the SDK does, it would be better to link out to the AWS documentation for the supported values: https://docs.aws.amazon.com/acm/latest/APIReference/API_CertificateDetail.html#ACM-Type-CertificateDetail-KeyAlgorithm

Reply:
thanks, @breathingdust, I've added an API reference.